### PR TITLE
Aerolyzer-12 Aerolyzer lint file

### PR DESCRIPTION
**Pull request by colliell**
Wiki has been updated

### CODE REVIEW

**Comment by colliell** [comment]
Accidentally closed then reopened! 
Using 4 spaces instead of tabs for indendation, max line length of 100, and checks import order using isort  (Pylint requires isort package by default, https://pypi.python.org/pypi/pylint/1.6.4)

File: aerolyzer_lint_file [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+# Tells whether to store unknown words to indicated private dictionary in
+# --spelling-private-dict-file option instead of raising a message.
+spelling-store-unknown-words=no
+
+
+[FORMAT]
+
+# Maximum number of characters on a single line.
+max-line-length=100
```
Comment (colliell) [inline aerolyzer_lint_file]: max line length set to 100

File: aerolyzer_lint_file [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+
+# List of optional constructs for which whitespace checking is disabled
+no-space-check=trailing-comma,dict-separator
+
+# Maximum number of lines in a module
+max-module-lines=1000
+
+# String used as indentation unit. This is usually " " (4 spaces) or "\t" (1
+# tab).
+indent-string='    '
```
Comment (colliell) [inline aerolyzer_lint_file]: 4 spaces instead of a tab

**Comment by kwhitehall** [comment]
Thanks @colliell. The travis error can be ignored and is b/c of the yaml file being worked on by @liusop. This LGTM. Merging. Thanks again!

**Pull request merged**